src/com/page/root/root.js

```diff
@@ -25,14 +25,32 @@ import PageDevPalette 					from 'com/page/dev/palette';
 
 import HeaderNoob						from 'com/header/noob/noob';
 
+const STORAGE_HEADER_DISMISSED = 'isNoobHeaderDismissed';
+
 export default class PageRoot extends Component {
+
+	constructor(props) {
+		super(props);
+		this.state = {
+			isNoobHeaderDisplayed: (!props.user || !props.user.id) &&
```

Comment:
Do we still need these first two checks? If we have it in local storage then why bother with the logic about the user being logged in or not?

Reply:
If we remove the first two checks then the popup will be visible for the users that are logged in until they close it. Logged-in users do not see the popup today so this would bring a bit of regression.

Reply:
True, though only until they dismiss it, then they'll never see it again. But it really doesn't matter so much, I'm just used to having cleaner render components (using react and redux) at work and I was hoping having the local storage flag be an excuse to remove the references to `user`, which I don't think the noob help text component has any business interacting with. But you are right that the feeling of regression might be too annoying.

Reply:
I definitely think that you've nailed down the philosophy of react/redux using lean components throughout the front-end code and I really like the component breakdown that you've done so far. Let me know if you'd rather go with the regression and I'll remove the user checks.

Cheers